tc-client/pom.xml

```diff
@@ -43,6 +43,10 @@
   </properties>
 
   <dependencies>
+    <dependency>
```

Comment:
This seems odd? Is it missing due to transitive exclusion now?

Reply:
You were quick ... I addressed this below.